<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirspeedConfig</class>
 <widget class="QWidget" name="AirspeedConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;h2&gt;Airspeed&lt;/h2&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../../qgroundcontrol.qrc">:/files/images/devices/BR-0004-03-2.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="enableCheckBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>67</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="useAirspeedCheckBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>108</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Airspeed</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="pinSpinBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>47</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>73</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pin Number</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="pinHelperGroupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>475</width>
     <height>118</height>
    </rect>
   </property>
   <property name="title">
    <string>Pin Selection Helper</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="hardwareGroupBox">
        <property name="title">
         <string>Choose Hardware</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="hardwareComboBox">
           <item>
            <property name="text">
             <string>Choose One</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>APM1.X</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>APM2.X</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PX4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pixhawk</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="sensorGroupBox">
        <property name="title">
         <string>Choose Sensor</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QComboBox" name="sensorComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="pinGroupBox">
        <property name="title">
         <string>Choose Pin</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QComboBox" name="pinComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../../qgroundcontrol.qrc"/>
 </resources>
 <connections/>
</ui>
